### ISSUE TITLE
Accessibility tests failing on advertising pages

### ISSUE THREAD

**Issue by stevejalim** [mozilla/bedrock#16796]
## Description

As per https://github.com/mozilla/bedrock/actions/runs/18380286076 there appear to be some a11y issues with the advertising page. We should fixup the markup or, if they're false positives, amend the tests.

**Comment by janbrasna** [comment]
#16787  "Aside should not be contained in another landmark"
